src/loaders/FileLoader.js

```diff
@@ -39,6 +39,19 @@ Object.assign( FileLoader.prototype, {
 
 		}
 
+		// If file is already in process of loading, wait for it to load.
+		if ( cached && cached.loaderSubscriptions ) {
```

Comment:
Can we replace `cached` with `cached !== undefined` here for the consistent code in the function?